### PR TITLE
Introduce simple editorconfig

### PR DESCRIPTION
**Pull request by paulcam206**
Add a simple `.editorconfig` to set indenting and charset standards across the codebase. Also, apply rules to existing files.

There will be a followup checkin later to introduce a `.clang-format` file for more in-depth style enforcement.

### CODE REVIEW

**Comment by khouzam** [comment]
> 		}

This still has tabs

---
Refers to: source/dotnet/Samples/XamarinFormsBotClient/AdaptiveCards.Sample.BotClient.Droid/Resources/Resource.Designer.cs:30 in 0a79ab2. [](commit_id = 0a79ab2e53c172597c3f06c3428287163e9e132b, deletion_comment = False)

File: source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/NumberInputRenderer.java [hunk 1]
```diff
@@ -77,6 +77,6 @@ else if ((numberInput = NumberInput.dynamic_cast(baseCardElement)) == null)
 
         return editText;
     }
-
```
Comment (RebeccaAnne) [inline source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/NumberInputRenderer.java]: Any idea what's causing this extra spacing to be added?

Comment (paulcam206) [reply source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/NumberInputRenderer.java]: It's changing from

```
<spaces><linefeed>
```

to 

```
<linefeed>
```

File: samples/Tests/InvalidMediaMix.json [hunk 2]
```diff
@@ -1 +1 @@
...
					"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"
				},
				{
					"mimeType": "audio/mpeg",
					"url": "https://adaptivecardsblob.blob.core.windows.net/assets/AdaptiveCardsOverviewVideo.mp4"
				}
			]
		}
	]
}
```
[the remark this answers is not in the document]
Comment (paulcam206) [reply samples/Tests/InvalidMediaMix.json]: This file had Mac-style linefeeds which apparently didn't agree with the tool I was using.

**Comment by paulcam206** [comment]
I blame the node.js tool that I was using (`eclint`). `.pfx` files should definitely not be affected by the `.editorconfig` since there are only explicit directives for common source filetypes.

**Pull request merged**